db/migrate/20221217172535_create_counties.rb

```diff
@@ -0,0 +1,10 @@
+class CreateCounties < ActiveRecord::Migration[7.0]
+  def change
+    create_table :counties do |t|
+      t.string :name
+      t.string :region
```

Comment:
Do we need indexes here?

Reply:
How does one decide whether an index is a good idea?   I'm not noticing any performance issues, but that is is a very controlled environment...

Reply:
Hm.  We do sort by region, and name within region.

Reply:
Added indices.